Source/Core/CesiumTerrainProvider.js

```diff
@@ -489,17 +493,16 @@ define([
      * @param {Number} x The X coordinate of the tile for which to request geometry.
      * @param {Number} y The Y coordinate of the tile for which to request geometry.
      * @param {Number} level The level of the tile for which to request geometry.
-     * @param {Boolean} [throttleRequests=true] True if the number of simultaneous requests should be limited,
-     *                  or false if the request should be initiated regardless of the number of requests
-     *                  already in progress.
+     * @param {Request} [request] The request object.
+     *
      * @returns {Promise.<TerrainData>|undefined} A promise for the requested geometry.  If this method
      *          returns undefined instead of a promise, it is an indication that too many requests are already
      *          pending and the request will be retried later.
      *
      * @exception {DeveloperError} This function must not be called before {@link CesiumTerrainProvider#ready}
      *            returns true.
      */
-    CesiumTerrainProvider.prototype.requestTileGeometry = function(x, y, level, throttleRequests) {
+    CesiumTerrainProvider.prototype.requestTileGeometry = function(x, y, level, request) {
```

Comment:
If HTTP/2 becomes widely adopted, I agree that throttling by server is a waste. However as of now throttling does make a big difference to performance:

I just got these numbers for an Everest static view:
```
4G with cache disabled:
  Throttled:
     9.70 seconds
     256 requests
  Not throttled:
     15.17 seconds
     412 requests

Normal usage
  Throttled
    1.45 seconds
    526 requests
  Not throttled
    1.60 seconds
    526 requests
```

When would the client not support HTTP/2?